### ISSUE TITLE
Wrong URL in DemoApp

### ISSUE THREAD

**Issue by erickaleida** [AvantiPoint/mobileauth-lib#10]
Struggled for over an hour until I realized the URL for the WebAuthenticator is wrong.  

In the code it is: 
Url = new Uri(new Uri(Constants.BaseUrl), $"mobileauth/{scheme}")

And it should be:
Url = new Uri(new Uri(Constants.BaseUrl), $"mobileauth/signin{scheme}")

**Comment by quintonv** [comment]
No wonder I had issues with this too when I tried. It just didn't work. Quite interesting how many sample/demo apps out there just don't work right out the box...

**Comment by chuckgiddens** [comment]
Yup, I fixed the same problem.  Not sure how this sample got published.

**Comment by chuckgiddens** [comment]
Using ngrok solves a lot of problems but I am getting this now:
![image](https://github.com/AvantiPoint/mobileauth-lib/assets/5237149/d230898c-30c9-4eef-a5d4-550028e39331)


Any ideas?
